governance.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Governance 1.0 DRAFT WIP
+description: Governance 1.0 discussion draft
+---
+# Governance 1.0 `DRAFT WIP`
+
+This document contains a work in progress (WIP) draft of the initial governance of the Op1st project (Governance 1.0).
```

Comment:
I would change Op1st to something like Operate First (Op1st) so that newcomers know what we're referring to

Reply:
are we a project or a community? I feel we can have multiple projects but there's one overarching community:

`... governance of the Operate First community`

Reply:
> are we a project or a community? I feel we can have multiple projects but there's one overarching community:
> 
> `... governance of the Operate First community`

This is a good question, as there may not be a formal definition of these things in open source software, but they are generally used in common ways:

* **Community** is the group of people sharing a common interest and desire to work toward something in common. This usage of the word "community" is different from its dictionary definition or use in other fields, and is more like a shorthand for _[community of practice](https://github.com/theopensourceway/guidebook/blob/main/communities_of_practice.adoc)_ (CoP).
*  **Project** is the collective term, referring to the community of people and the formal structure the community creates and sustains in order to get their work done; a project is defined by a vision and driven by a mission, which are created and brought to life by the community.
* **Subproject** is typically a formal structure within a project, either organized at the top or within a SIG. This is the one that sounds most like what you describe above has "(having) multiple projects".

Organizations that contain multiple projects (as per the above usage) and are overarching are more like foundations, e.g. Python Software Foundation, Apache Software Foundation, and Cloud Native Computing Foundation.

So for example, if you are thinking of communities like OpenData Hub and Open Cluster Manager, they are forming into projects we are loosely affiliated with, but not even a loose confederation exists.You can have groups of projects that coordinate together without a formal structure.

Seeing all the +1s to your comment, what I'll do is add some clarifying language, and see if that suffices. Otherwise, we can start with "community" and do any changes as part of the 1.1 updates.

Reply:
> I would change Op1st to something like Operate First (Op1st) so that newcomers know what we're referring to

Thank you, yes, this was a very sloppy first draft paragraph. I will expand it; but as per #104 to use 'Op1st' for the community name, my proposal is that Op1st is no longer a shorthand for Operate First but rather the distinctly separate name for the Op1st community, which is implementing an Operate First envirionment.






























































































```

Reply:
Not sure I'm convinced about the `op1st` thing. But let's have that discussion in #104 :)

Reply:
```suggestion
This document contains a work in progress (WIP) draft of the initial governance (Governance 1.0) of the community-formed project around the _[Operate First concept](https://openinfralabs.org/operate-first-manifesto/)_.
In this document, the community and project are referred to by the shorthand _Op1st_ (spoken as "op first").
```

The above language is intended to not-yet name the community ("Operate First" or "not-Operate First"). The explanation let's us use the shorthand throughout the document here, which is easy to change after we have a governance to enable us to discuss and #104 as a community, while not pre-deciding a community/project name.

Reply:
This looks good to me!